mlflow-tests/tests/test_artifacts.py

```
@@ -0,0 +1,242 @@
+            # Step 2: Create user with permissions
+            logger.info(f"Step 2: Creating user with {verb_names} permissions on {[rt.value for rt in test_data.user_info.resource_types]} in workspace '{test_data.user_info.workspace}'")
+            user_info: UserInfo = create_user_with_permissions(
+                workspace=test_data.user_info.workspace,
+                verbs=test_data.user_info.verbs,
+                resource_types=test_data.user_info.resource_types,
+                subresources=test_data.user_info.subresources
+            )
+            logger.info(f"Created user: {user_info.uname}")
+
+            # Step 3: Set test context and workspace
+            logger.debug(f"Step 3: Setting active user and workspace context")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)